README.developer.md

```diff
@@ -333,6 +333,8 @@ The package build process can be adjusted using following environment variables:
   - To skip installation of build dependencies this option should contain `yes` value.
 - `WITH_PYTHON`
   - To skip building python bluechi modules this option should contain `0`.
+- `NO_COVERAGE`
+  - To start collecting coverage this option should contain `0`.
```

Comment:
We don't want to build with code coverage by default, so using `WITH_COVERAGE` option makes more sense to me